src/port/switch/SwitchImpl.cpp

```diff
@@ -74,6 +76,51 @@ void Ship::Switch::SetupFont(ImFontAtlas* fonts) {
     plExit();
 }
 
+void Ship::Switch::ImGuiSwapABXY(int startEvent) {
+    if (CVarGetInteger("gOpenMenuBar", 0)) {
+        return;
+    }
```

Comment:
this part makes no sense to me. in my testing, it makes it so we aren't doing the button switching at all when using the imgui menus. This means we have the switch "B" button acting as "confirm" and the switch "A" button acting as "cancel"

Reply:
to add to this, I tried commenting this block out and I'm running into the issue where I can only press a button once when navigating the menu before input stops working and the controller can no longer be used to navigate the menu